composer.json

```diff
@@ -27,13 +27,13 @@
         }
     },
     "require": {
-        "php": "^8.0",
-        "laravel/framework": "^9.0"
+        "php": "^8.1",
+        "laravel/framework": "^10.0"
     },
     "require-dev": {
         "ext-pdo": "*",
-        "orchestra/testbench": "^7.0",
-        "phpunit/phpunit": "^9.5.10"
+        "orchestra/testbench": "^8.0",
+        "phpunit/phpunit": "^9.6.0"
```

Comment:
PHP Unit should be at least version ^10.0 because laravel/framework use version ^10.0

Ref: [Laravel Upgrade Docs](https://laravel.com/docs/10.x/upgrade)